projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.html

```diff
@@ -51,8 +53,13 @@ <h3>{{"SUBTASKS"|translate}}</h3>
             </div>
         }
     </div>
-    <div class="add-files-button" [ngClass]="{'add-files-button-sub': task.children.length === 0}">
-        <button mat-flat-button color="primary" (click)="addFiles()">{{ 'ADD_FILES' | translate }}</button>
+    <div class="add-files-button flex gap-x-3" [ngClass]="{'add-files-button-sub': task.children.length === 0}">
+        <button mat-flat-button class="primary-button" (click)="addFiles()">{{ 'ADD_FILES' | translate }}</button>
+        <ng-container *ngIf="task.learningResources && task.learningResources.length > 0 ">
```

Comment:
See here also use @if do check the whole file once before pushing it

Reply:
done